cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3205,9 +3286,12 @@ request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_cisco_ios` sourcetype=cisco:ios "%AAA-I-CONNECT" \
 | rex "CONNECT:\s(?<message>.*)" \
-| table _time message \
+| table _time dvc message \
 | `cs_cisco_ios_new_connection_for_user_filter`
 action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.contributing_events = `cs_cisco_ios` sourcetype=cisco:ios "%AAA-I-CONNECT"
```

Comment:
Please check to see if we have user or source_ip information to be used as a attacker.

Reply:
Currently, the src_ip and user_name fields are not getting extracted.

Reply:
add src field in the attacker search as user is not present in all the events.